src/AutomationTests/SnapshotCommandUnitTests.cs

```diff
@@ -177,11 +117,13 @@ public void Execute_UnableToSelectCandidateElement_ThrowsAutomationException_Err
                     Select = () => false,
                 };
 
-                InitializeShims(selectAction:selectAction, populateLocationHelper: false, enableRetention: true, shimTargetElementLocator: true);
+                InitializeShims(selectAction:selectAction, shimTargetElementLocator: true);
+
+                var config = Config.Builder.ForProcessId(-1).Build();
 
                 try
                 {
-                    SnapshotCommandResult result = SnapshotCommand.Execute(new Dictionary<string, string>());
+                    SnapshotCommandResult result = SnapshotCommand.Execute(config, OutputFileHelperStub);
                 }
                 catch (AxeWindowsAutomationException ex)
```

Comment:
>catch [](start = 16, length = 5)

Many of these tests look like they should fail if no Exception is thrown, but there's no enforcement